preprocessor.py

```diff
@@ -1,11 +1,299 @@
 # coding: utf-8
 
 
+import numpy as np
+
 from keras.applications.imagenet_utils import preprocess_input
-from keras.preprocessing.image import img_to_array
+from keras.preprocessing.image import (img_to_array,
+                                       apply_affine_transform,
+                                       apply_channel_shift,
+                                       apply_brightness_shift)
+
+
+class PreProcessor(object):
+    """PreProcessor
+    Referred from keras ImageDataGenerator.
```

Comment:
This `PreProcessor` is inspired from `ImageDataGenerator`.
https://github.com/keras-team/keras/blob/master/keras/preprocessing/image.py#L233